index.html

```diff
@@ -1,40 +1,70 @@
 <html>
     <head>
         <title>Tipease II</title>
+        
```

Comment:
Consider the addition of a responsive meta tag in this general region.

Reply:
Thanks for the reminder!

Reply:
👍